modules/iam-role-for-service-accounts-eks/variables.tf

```
@@ -158,6 +158,12 @@ variable "mountpoint_s3_csi_bucket_arns" {
   default     = []
 }
 
+variable "mountpoint_s3_csi_kms_cmk_ids" {
```

Lets drop the `cmk` from the variable name

```suggestion
variable "mountpoint_s3_csi_kms_ids" {
```

---

okay, I will do that.
I was copying the name of the ebs policy, where `ebs_csi_kms_cmk_ids` is used as variable name.